### PR TITLE
Update code snippets for PDF/A and printing sections

### PR DESCRIPTION
**Pull request by BaronRastignak**

### CODE REVIEW

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 1]
```diff
@@ -83,13 +83,64 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
+        pgs.PaperSize = Aspose.Pdf.Printing.PaperSizes.A4;
+
+        // Set PageMargins (if required)
+        pgs.Margins = new Aspose.Pdf.Devices.Margins(0, 0, 0, 0);
+
+        // Print document using printer and page settings
+        viewer.PrintDocumentWithSettings(pgs, ps);
+
+        // Close the PDF file after printing
+        viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 2]
```diff
@@ -100,56 +151,174 @@ public static void SimplePrint()
...
     // Set PageMargins (if required)
-    pgs.Margins = new System.Drawing.Printing.Margins(0, 0, 0, 0);
+    pgs.Margins = new Aspose.Pdf.Devices.Margins(0, 0, 0, 0);
 
     // Print document using printer and page settings
     viewer.PrintDocumentWithSettings(pgs, ps);
 
-    // Close the PDF file after priting
+    // Close the PDF file after printing
     viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/print-pdf-document-in-wpf-application/_index.md [hunk 3]
```diff
@@ -139,37 +221,103 @@ In this case, we will follow these steps:
...
-        var xpsDoc = new XpsDocument(package, CompressionOption.Maximum, inMemoryPackageName);
-        DocViewer.Document = xpsDoc.GetFixedDocumentSequence();
+        // Open the XPS document from the package
+        using var xpsDoc = new XpsDocument(package, CompressionOption.Maximum, inMemoryPackageName);
+
+        // Display the document in the DocumentViewer
+        docViewer.Document = xpsDoc.GetFixedDocumentSequence();
+
+        // Close the XPS document after use
         xpsDoc.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-pdf-document-in-wpf-application/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 4]
```diff
@@ -100,56 +151,174 @@ public static void SimplePrint()
...
+            // using Aspose.Pdf.Printing.Extensions;
+            Aspose.Pdf.Printing.PrinterSettings ps = printDialog.PrinterSettings.ToAsposePrinterSettings();
+            Aspose.Pdf.Printing.PageSettings pgs = printDialog.PrinterSettings.DefaultPageSettings.ToAsposePageSettings();
+
+            // Print document using printer and page settings
+            viewer.PrintDocumentWithSettings(pgs, ps);
+        }
+
+        // Close the PDF file after printing
+        viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 5]
```diff
@@ -100,56 +151,174 @@ public static void SimplePrint()
...
+        // using Aspose.Pdf.Printing.Extensions;
+        Aspose.Pdf.Printing.PrinterSettings ps = printDialog.PrinterSettings.ToAsposePrinterSettings();
+        Aspose.Pdf.Printing.PageSettings pgs = printDialog.PrinterSettings.DefaultPageSettings.ToAsposePageSettings();
+
+        // Print document using printer and page settings
+        viewer.PrintDocumentWithSettings(pgs, ps);
+    }
+
+    // Close the PDF file after printing
+    viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/printing-pdf-to-an-xps-printer-facades/_index.md [hunk 6]
```diff
@@ -81,15 +81,67 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
+        pgs.PaperSize = Aspose.Pdf.Printing.PaperSizes.A4;
+
+        // Set PageMargins (if required)
+        pgs.Margins = new Aspose.Pdf.Devices.Margins(0, 0, 0, 0);
+
+        // Print document using printer and page settings
+        viewer.PrintDocumentWithSettings(pgs, ps);
+
+        // Close the PDF file after printing
+        viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-to-an-xps-printer-facades/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/printing-pdf-to-an-xps-printer-facades/_index.md [hunk 7]
```diff
@@ -100,44 +152,86 @@ public static void PrintToXpsPrinter()
...
     // Set PageMargins (if required)
-    pgs.Margins = new System.Drawing.Printing.Margins(0, 0, 0, 0);
+    pgs.Margins = new Aspose.Pdf.Devices.Margins(0, 0, 0, 0);
 
     // Print document using printer and page settings
     viewer.PrintDocumentWithSettings(pgs, ps);
 
-    // Close the PDF file after priting
+    // Close the PDF file after printing
     viewer.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-to-an-xps-printer-facades/_index.md]: No, it's closed on Dispose. Removed the line

File: net/converting/convert-pdfa-to-pdf/_index.md [hunk 8]
```diff
@@ -83,28 +83,94 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
 ```csharp
-public static void ConvertPDFAtoPDF()
+// For complete examples and data files, visit https://github.com/aspose-pdf/Aspose.PDF-for-.NET
+
+private static void ConvertPDFAtoPDF()
 {
-    // For complete examples and data files, please go to https://github.com/aspose-pdf/Aspose.PDF-for-.NET
-    Document document = new Document(dataDir + "PDFAToPDF.pdf");
+    // The path to the documents directory.
+    string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/converting/convert-pdfa-to-pdf/_index.md:94]: I add an empty line before each block of code beginning with a comment, to improve readability of snippets

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 9]
```diff
@@ -100,56 +151,174 @@ public static void SimplePrint()
...
+private static void PrintWithPrintDialog()
+{
+    // The path to the documents directory.
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create PdfViewer object
+    using var viewer = new Aspose.Pdf.Facades.PdfViewer();
+
+    // Open input PDF file
+    viewer.BindPdf(dataDir + "input.pdf");
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: Fixed

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 10]
```diff
@@ -83,13 +83,64 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
+private static void SimplePrint()
+{
+    // The path to the documents directory.
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create PdfViewer object
+    using var viewer = new Aspose.Pdf.Facades.PdfViewer();
 
     // Open input PDF file
     viewer.BindPdf(dataDir + "input.pdf");
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: Fixed

File: net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md [hunk 11]
```diff
@@ -83,13 +83,64 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
 {
+    // The path to the documents directory.
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
     // Create PdfViewer object
-    PdfViewer viewer = new PdfViewer();
+    using (var viewer = new Aspose.Pdf.Facades.PdfViewer())
+    {
+        // Open input PDF file
+        viewer.BindPdf(dataDir + "input.pdf");
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/printing-pdf-in-net-framework/_index.md]: Fixed

File: net/advanced-operations/printing/print-pdf-document-in-wpf-application/_index.md [hunk 12]
```diff
@@ -84,48 +84,130 @@ The following code snippet also work with [Aspose.PDF.Drawing](/pdf/net/drawing/
...
+                            {
+                                // Print converted document
+                                printDialog.PrintDocument(fixedDocumentSequence.DocumentPaginator,
+                                    "A fixed document");
+                            }
+                        }
+        
+                        // Remove the package from the store and close the document after the print
+                        PackageStore.RemovePackage(packageUri);
+                        xpsDoc.Close();
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-pdf-document-in-wpf-application/_index.md]: No, it's closed on Dispose. Removed the line

File: net/advanced-operations/printing/print-dotnetcore/_index.md [hunk 13]
```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
...
+// For complete examples and data files, visit https://github.com/aspose-pdf/Aspose.PDF-for-.NET
+
+private static void PrintWithNetCore()
+{
+    // The path to the documents directory.
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create the secondary thread and pass the printing method for
+    // the constructor's ThreadStart delegate parameter.
+    var printingThread = new Thread(() => PrintPDF(dataDir + "input.pdf"));
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md]: Fixed

File: net/advanced-operations/printing/print-dotnetcore/_index.md [hunk 14]
```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
...
+// For complete examples and data files, visit https://github.com/aspose-pdf/Aspose.PDF-for-.NET
+
+private static void PrintWithNetCore()
 {
-    static void Main()
-    {
-        // Create the secondary thread and pass the printing method for
-        // the constructor's ThreadStart delegate parameter.
-        Thread printingThread = new Thread(() => PrintPDF(@"C:\tmp\doc-pdf.pdf"));
+    // The path to the documents directory.
```
[the remark this answers is not in the document]
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md]: Fixed

File: net/advanced-operations/printing/print-dotnetcore/_index.md [hunk 15]
```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
...
-    Page page = document.Pages.Add();
-    page.Paragraphs.Add(new TextFragment("Hello world!"));
+    // The path to the documents directory
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create a new document
+    using (var document = new Aspose.Pdf.Document())
+    {
+        // Add a new page and some content
+        Aspose.Pdf.Page page = document.Pages.Add();
```
Comment (AlexBiriukov) [inline net/advanced-operations/printing/print-dotnetcore/_index.md:200]: Use var on left

File: net/advanced-operations/printing/print-dotnetcore/_index.md [hunk 16]
```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
...
+private static void PickTrayByPdfSize()
+{
+    // The path to the documents directory
+    string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Printing();
+
+    // Create a new document
+    using var document = new Aspose.Pdf.Document();
+
+    // Add a new page and some content
+    Aspose.Pdf.Page page = document.Pages.Add();
```
Comment (AlexBiriukov) [inline net/advanced-operations/printing/print-dotnetcore/_index.md:226]: Use var on left

File: net/advanced-operations/printing/print-dotnetcore/_index.md [hunk 17]
```diff
@@ -85,102 +85,215 @@ The Aspose.PDF library allows us to convert PDF files to XPS. This function can
...
+{{< /tabs >}}
 
 ## Print Dialog Presets Page Scaling
 
 The next code snippet is intended to ensure that the [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property is correctly applied and saved in the PDF.
 
-The [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property has been added to the [Document](https://reference.aspose.com/pdf/net/aspose.pdf/document/) class with values ​​Aspose.Pdf.PrintScaling.AppDefault or Aspose.Pdf.PrintScaling.None.
+The [PrintScaling](https://reference.aspose.com/pdf/net/aspose.pdf/document/printscaling/) property has been added to the [Document](https://reference.aspose.com/pdf/net/aspose.pdf/document/) class with values `​​Aspose.Pdf.PrintScaling.AppDefault` or `Aspose.Pdf.PrintScaling.None`.
+
+The page scaling option that shall be selected when a print dialog is displayed for this document. Valid values are `None`, which indicates no page scaling, and `AppDefault`, which indicates the conforming reader's default print scaling. If this entry has an unrecognized value, `AppDefault` should be used. Default value: `AppDefault`.
```
Comment (AnHolub) [inline net/advanced-operations/printing/print-dotnetcore/_index.md:245]: The page scaling option shall be selected when a print dialog is displayed for this document

**Review by AnHolub — APPROVED** [review]

File: net/advanced-operations/printing/print-dotnetcore/_index.md (hunk 15 above)
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md]: Fixed

File: net/advanced-operations/printing/print-dotnetcore/_index.md (hunk 17 above)
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md:245]: Fixed

File: net/advanced-operations/printing/print-dotnetcore/_index.md (hunk 16 above)
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md]: Fixed

File: net/advanced-operations/printing/print-dotnetcore/_index.md (hunk 17 above)
Comment (BaronRastignak) [reply net/advanced-operations/printing/print-dotnetcore/_index.md:245]: It's from the PDF specification. This option controls the behavior of a print dialog, for example, in the Adobe Acrobat: if a dialog has an option "Print Scaling", what value will be selected there by default

**Pull request merged**